### PR TITLE
Update info in slack welcome message

### PR DESCRIPTION
**Pull request by marianabocoi**
Ref. #2335

Seems like there are some updates needed to the welcome message.

/cc @mrbobbytables

### CODE REVIEW

**Comment by ameukam** [comment]
/ok-to-test
/lgtm
/approve
Thank you so much for doing this!

/hold
Remove hold when someone can deploy.

**Comment by nikhita** [comment]
The slack guidelines can be updated to https://www.kubernetes.dev/docs/comms/slack/#escalating-andor-reporting-a-problem

/assign @mrbobbytables 
for lgtm from contribex

**Comment by marianabocoi** [comment]
> The slack guidelines link can be updated to https://www.k8s.dev/docs/comms/slack/#escalating-andor-reporting-a-problem

@nikhita Fixed, thanks for checking! ❤️

File: apps/slack-infra/resources/slack-welcomer/message.yaml [hunk 1]
```diff
@@ -16,13 +16,13 @@ data:
...
     *Looking to contribute?*
-    - Ask questions in  #kubernetes-dev.
+    - Ask questions in  #kubernetes-contributors.
     - Join a <https://git.k8s.io/community/sig-list.md|SIG channel>.
     - Have your question answered live  on stream in #meet-our-contributors during the monthly <https://git.k8s.io/community/meet-our-contributors.md|MoC>.
     - Have a PR you need reviewed? Hop into #pr-reviews to get some feedback.
 
     *See some spam? or being harrassed?*
-    - Use the _"<https://git.k8s.io/community/communication/slack-guidelines.md#escalating-andor-reporting-a-problem|Report Message>"_ message action.
+    - Use the _"< https://www.k8s.dev/docs/comms/slack/#escalating-andor-reporting-a-problem|Report Message>"_ message action.
```
Comment (nikhita) [inline apps/slack-infra/resources/slack-welcomer/message.yaml:25]:
```suggestion
    - Use the _"<https://www.k8s.dev/docs/comms/slack/#escalating-andor-reporting-a-problem|Report Message>"_ message action.
```

**Review by nikhita — COMMENTED** [review]
Small nit but looks great!

File: apps/slack-infra/resources/slack-welcomer/message.yaml (hunk 1 above)
Comment (marianabocoi) [reply apps/slack-infra/resources/slack-welcomer/message.yaml:25]: 🙈

**Review by mrbobbytables — COMMENTED** [review]
One other small change if you don't mind updating it (I can't select the line in review itself)

On like 35, theres another link to the slack guidelines
```
Have any other questions? Please see the <http://git.k8s.io/community/communication/slack-guidelines.md|Slack communications guidelines> to start.
```

Could you update that one to route folks to the k8s.dev as well?
Figure while we're editing the file, might as well take care of everything if possible :)

**Comment by marianabocoi** [comment]
I think I got all of them now I finally understood what you meant 😄 🎉 
@nikhita @mrbobbytables I hope I did not miss anything this time 🙈

**Review by mrbobbytables — COMMENTED** [review]
Thanks!
/lgtm

**Comment by ameukam** [comment]
/hold cancel
I'll take care of the deploy.

**Comment by ameukam** [comment]
Deployment is done. 

```shell
$ ./deploy.sh
managedcertificate.networking.gke.io/slack-infra unchanged
Warning: networking.k8s.io/v1beta1 Ingress is deprecated in v1.19+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
ingress.networking.k8s.io/slack-infra configured
deployment.apps/slack-event-log unchanged
service/slack-event-log unchanged
deployment.apps/slack-moderator unchanged
service/slack-moderator unchanged
deployment.apps/slack-moderator-words unchanged
configmap/slack-moderator-words-filters unchanged
service/slack-moderator-words unchanged
deployment.apps/slack-post-message unchanged
service/slack-post-message unchanged
deployment.apps/slack-welcomer unchanged
configmap/slack-welcomer-message configured
service/slack-welcomer unchanged
configmap/bad-domains unchanged
deployment.apps/slackin-deployment unchanged
service/slackin2 unchanged
secret/recaptcha unchanged
secret/slack-event-log-config unchanged
secret/slack-moderator-config unchanged
secret/slack-moderator-words-config configured
secret/slack-welcomer-config unchanged
secret/slackin-token unchanged
```

**Pull request merged**